.gitignore

```diff
@@ -3,6 +3,7 @@
 .bundle
 .config
 .yardoc
+.env
```

Comment:
This is here because we never want an `.env` file to be committed to this repo, as the presence of one might interfere with this gem's intended behaviour of relying on whatever app it's installed in for environment data.